lib/AST/ASTVerifier.cpp

```diff
@@ -2076,7 +2076,7 @@ class Verifier : public ASTWalker {
 
       // Variables must have materializable type, unless they are parameters,
       // in which case they must either have l-value type or be anonymous.
-      if (!var->getInterfaceType()->isMaterializable()) {
+      if (!var->getInterfaceType()->isMaterializable() || var->isInOut()) {
```

Comment:
Presumably this is pre-emptive for when `getInterfaceType()` is eventually updated to not add the `InOutType` back?